redhat-cop/casl-ansible
# Fix deprication warnings with ansible 2.4 for casl-ansible

#### What does this PR do?
Fixing Ansible 2.4 deprecated warnings

#### How should this be manually tested?
Use the `sample` inventories with the `openshift-applier` test(s) or the `openshift-cluster-seed` playbook. Make sure to exercise both local and non-local hosts.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
